src/lib/y2firewall/widgets/pages/zones.rb

```diff
@@ -79,7 +79,6 @@ def handle
             result = Dialogs::Zone.run(zone, new_zone:       true,
                                              existing_names: firewall.zones.map(&:name))
             if result == :ok
-              zone.relations.map { |r| zone.send("#{r}=", []) }
```

Comment:
what is goal of this change? commit message says what you did, but not why.

Reply:
You are right, I should mention in the changelog or somewhere. 

So this initialization was probably needed before added it to the constructor, which was later done (later taking in account the date it was added here).

https://github.com/yast/yast-firewall/commit/873bbf5249d1cbeb13b9db981da948cf605f6689#diff-957421c569fb30a22c554130ec95497eR72

https://github.com/yast/yast-yast2/pull/840/files#diff-5f18f6aba33e047bd40791544760f288R65

Reply:
ok, that makes sense.